### PR TITLE
Update gtfs.markdown

### PR DESCRIPTION
**Pull request by Friedrieck**
To clarify what is expected for the offset parameter (based on the error message I got in the log).

## Proposed change
I had some pain to set the `offset` parameter, so I hope my addition (inspired from other integration docs, e.g. for `scan_interval`), will help others.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Friedrieck, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#21670).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/gtfs.markdown [hunk 1]
```diff
@@ -62,10 +62,10 @@ name:
   default: GTFS Sensor
   type: string
 offset:
-  description: A minimum delay to look for. If a departure is in less time than `offset`, it will be ignored.
+  description: A minimum delay to look for. If a departure is in less time than `offset`, it will be ignored. Supported `time` formats: `offset: 'HH:MM:SS'`, `offset: 'HH:MM'`. 
   required: false
   default: 0
-  type: [integer, time]
+  type: [integer (seconds), time]
```
Comment (frenck) [inline source/_integrations/gtfs.markdown]: "integer (seconds)" is not a valid data type.
Please revert this change

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/gtfs.markdown (hunk 1 above)
Comment (Friedrieck) [reply source/_integrations/gtfs.markdown]: Done.

**Review by frenck — DISMISSED** [review]
Thanks, @Friedrieck 👍

File: source/_integrations/gtfs.markdown [hunk 2]
```diff
@@ -62,7 +62,7 @@ name:
   default: GTFS Sensor
   type: string
 offset:
-  description: A minimum delay to look for. If a departure is in less time than `offset`, it will be ignored.
+  description: A minimum delay to look for. If a departure is in less time than `offset`, it will be ignored. Set the value either in seconds (`integer`), or using one of these `time` formats: `offset: 'HH:MM:SS'`, `offset: 'HH:MM'`.
```
Comment (frenck) [inline source/_integrations/gtfs.markdown]:
```suggestion
  description: "A minimum delay to look for. If a departure is in less time than `offset`, it will be ignored. Set the value either in seconds (`integer`), or using one of these `time` formats: `offset: 'HH:MM:SS'`, `offset: 'HH:MM'`. "
```

**Review by frenck — APPROVED** [review]
